server/manager/src/main/java/org/apache/accumulo/master/tableOps/namespace/rename/RenameNamespace.java

```diff
@@ -62,15 +64,15 @@ public RenameNamespace(NamespaceId namespaceId, String oldName, String newName)
       final String tap = master.getZooKeeperRoot() + Constants.ZNAMESPACES + "/" + namespaceId
           + Constants.ZNAMESPACE_NAME;
 
-      zoo.mutate(tap, null, null, current -> {
-        final String currentName = new String(current);
+      zoo.mutateExisting(tap, current -> {
+        final String currentName = new String(current, UTF_8);
```

Comment:
Nice catch.  Yuck, this should have been using UTF8.  If the current persisted name is encoded w/ something else it likely would not matter w/ the restrictions on what characters could go into a namespace.